### PR TITLE
Adding RediSearch build script and Dockerfiles

### PR DESCRIPTION
**Pull request by kandarpamalipeddi**

### CODE REVIEW

File: r/redisearch/Dockerfiles/2.0.0-ubi8/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,48 @@
+FROM registry.access.redhat.com/ubi8/ubi AS builder
```
Comment (seth-priya) [inline r/redisearch/Dockerfiles/2.0.0-ubi8/Dockerfile:1]: let's explicitly add an attribution stating that this Dockerfile is based on and derived from the community versions at <GitHub URL>

File: r/redisearch/redisearch_rhel8.sh [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+ 
+make c_tests
+make pytest
+################################################################################
+# Note: 
+# Currently observing 2 test failures with cpp_tests and goes into hang state.
+# Hence, this script need to terminate manually.
+# Soon, going to publish the patch to fix this.
+################################################################################
+make cpp_tests
```
Comment (seth-priya) [inline r/redisearch/redisearch_rhel8.sh:60]: lets disable the text execution till we can have patch uploaded (post required clearance) or up-streamed.

**Review by seth-priya — CHANGES_REQUESTED** [review]

File: r/redisearch/redisearch_rhel8.sh (hunk 2 above)
Comment (kandarpamalipeddi) [reply r/redisearch/redisearch_rhel8.sh]: Changes made as per your suggestion.

File: r/redisearch/Dockerfiles/2.0.0-ubi8/Dockerfile (hunk 1 above)
Comment (kandarpamalipeddi) [reply r/redisearch/Dockerfiles/2.0.0-ubi8/Dockerfile:4]: Changes made as per your suggestion.

**Review by seth-priya — APPROVED** [review]

**Pull request merged**